### ISSUE TITLE
bean-report ledger price statements slightly misaligned with quoted commodities

### ISSUE THREAD

**Issue by blais** [beancount/beancount#232]
**[Original report](https://bitbucket.org/blais/beancount/issue/232) by Martin Michlmayr (Bitbucket: [tbm13](https://bitbucket.org/tbm13), GitHub: [tbm](https://github.com/tbm)).**

----------------------------------------

bean-report ledger doesn't format price statements correctly for quoted 
commodities because commodities are quoted after the formating takes
place.

```
P 2017/12/31 00:00:00 "GB00B80QGD89"            6.353 GBP

P 2017/12/31 00:00:00 "GB00B80QFW04"            3.608 GBP

P 2018/03/14 00:00:00 EUR                      0.89 GBP
```

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

By "correctly" here you mean, aligned columns?

**Comment by blais** [comment]
**Original comment by Martin Michlmayr (Bitbucket: [tbm13](https://bitbucket.org/tbm13), GitHub: [tbm](https://github.com/tbm)).**

----------------------------------------

Yes, sorry, it's just the minor misalignment I was talking about.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Merged in tbm13/beancount/bug232 (pull request #54)

Fixed #232: Align quoted commodities correctly in ledger price statements

Approved-by: Martin Blais <blais@furius.ca>

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #232: Align quoted commodities correctly in ledger price statements

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Merged in tbm13/beancount/bug232 (pull request #54)

Fixed #232: Align quoted commodities correctly in ledger price statements

Approved-by: Martin Blais <blais@furius.ca>
